src/vm/trace_context.zig

```diff
@@ -0,0 +1,132 @@
+const std = @import("std");
```

Comment:
please add the new `trace_context` file declaration in `lib.zig` for usage as lib as well as automatic fetching of unit tests, and generation of docs